README.md

```diff
@@ -1788,6 +1789,43 @@ Map::from( ['a' => 'X', 'b' => 'Y'] )->has( 'X' );
 ```
 
 
+### if()
+
+Executes callables $then and $else depending on callable $condition.
+
+```php
+public function if( Callable $condition, Callable $then, $Callable $else = null ) : bool
+```
+* @param callable $condition Function with (Map) parameter and returns bool
```

Comment:
Type hint must be `callable|bool`. Also, the description should state that it must evaluate to bool, not return bool

Reply:
fixed in c329afdda114015b900470c1ef73edc6439fc1a1